core/forms.py

```diff
@@ -44,10 +45,49 @@ def __init__(self: "Post", *args: tuple, **kwargs: dict) -> None:
 class CommunityForm(forms.ModelForm):
     class Meta:
         model = Community
-        fields: typing.ClassVar = ["name"]
+        fields: typing.ClassVar = ["name", "privacy", "is_18_plus"]
 
     def __init__(self: "CommunityForm", *args: list, **kwargs: dict) -> None:
         super().__init__(*args, **kwargs)
         self.helper = FormHelper()
         self.helper.form_method = "post"
         self.helper.add_input(Submit("submit", "Create Community"))
+
+        self.fields["privacy"].widget = forms.RadioSelect(choices=Community.PRIVACY_CHOICES)
+        self.fields["privacy"].initial = "10_PUBLIC"
```

Comment:
In model default privacy is restricted, here is public. It is inconsistent. Also, when setting default on model, it should  show default on the form? So this 56 and 57 line is not needed.